.travis.yml

```diff
@@ -156,18 +145,12 @@ jobs:
             - $GENERATED_APPIMAGE
             - $APPIMAGE_CHECKSUM
             - $TAR_CHECKSUM
-          skip_cleanup: true
```

Comment:
why remove this?

Reply:
This is now the default behavior, and this flag has been deprecated. There is a new flag if you want to clean up.